Add open in browser feature

richiksc opened the pull request:
Adds a simple 'open in default browser' feature for the built HTML file by passing the `--open` or `-o` flag.
I'm a Rust newbie, so feel free to change the code.

richiksc committed:
Add open in browser feature

dominikWin commented:
Thanks for working on this! Only thing I see is that this uses eprintln. Everything else uses the error and warning until macros. If you update that I'll accept the request. I'm not sure if error or warning is better here though

richiksc committed:
Change eprintln -> warning

I added another brace block to the warning macro
because I was getting a compile error without them.

richiksc commented:
Converted `eprintln` to `warning`. I added another set of curly braces because I was getting a compile error without them.

dominikWin commented:
Thanks!
